<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="fr" sourcelanguage="en">
<context>
    <name>generals</name>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="44"/>
        <source>Show errors</source>
        <translation>Afficher erreures</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="45"/>
        <source>Show warnings</source>
        <translation>Afficher avertissements</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="36"/>
        <source>Create layer</source>
        <translation>Créer layer</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="39"/>
        <source>Cancel</source>
        <translation>Annuler</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="42"/>
        <source>Show error log</source>
        <translation>Afficher le log des erreurs</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="43"/>
        <source>Upload xtf-log file</source>
        <translation>Choisir chemin ou URL d'un fichier XTF-Error-Log</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="44"/>
        <source>Show log for existing layer</source>
        <translation>Afficher le log des erreurs pour un layer existant</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="45"/>
        <source>Only layers created with this plugin can be selected</source>
        <translation>Seuls les layers créés avec ce plugin peuvent être sélectionnés</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="252"/>
        <source>No valid file</source>
        <translation>Fichier non valide</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="242"/>
        <source>Could not get a valid XTF-Log file from specified Url</source>
        <translation>Impossible d'obtenir un fichier XTF-Log valide à partir de l'URL spécifiée</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="252"/>
        <source>No valid XTF-Log file at specified Path</source>
        <translation>Aucun fichier XTF-Log valide au chemin d'accès spécifié</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dock_panel.py" line="42"/>
        <source>Error log</source>
        <translation>XTFLog-Checker log des erreurs</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="388"/>
        <source>No Errors</source>
        <translation>Aucune erreur</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="185"/>
        <source>The selected XTF file contains no Ilivalidator-Errors, select another file.</source>
        <translation>Le fichier sélectionné ne contient aucune erreur du type IliVErrors, sélectionnez un autre fichier.</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="247"/>
        <source>Large file</source>
        <translation>Fichier de plus de 5 MB</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="247"/>
        <source>Processing of large XTF-Log files might take a while</source>
        <translation>Le traitement de fichiers volumineux peut prendre un certain temps</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="100"/>
        <source>Unsupported file</source>
        <translation>Fichier non pris en charge</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="100"/>
        <source>Model {model_name} not supported.</source>
        <translation>Modèle non pris en charge</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="282"/>
        <source>No valid geometry</source>
        <translation>Aucune géométrie valide</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="282"/>
        <source>No Point, Line or Surface Geometries found.</source>
        <translation>Aucune géométrie de point, ligne ou surface trouvée</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_dialog.py" line="388"/>
        <source>The selected XTF file contains no igCheck-Errors, select another file.</source>
        <translation>Le fichier XTF sélectionné ne contient aucune erreur igCheck, veuillez en sélectionner un autre</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="35"/>
        <source>Show infos</source>
        <translation>Afficher les infos</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="57"/>
        <source>igCheck - Point Errors</source>
        <translation>igCheck – Erreurs de points</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="59"/>
        <source>igCheck - Line Errors</source>
        <translation>igCheck – Erreurs de lignes</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="61"/>
        <source>igCheck - Surface Errors</source>
        <translation>igCheck – Erreurs de surfaces</translation>
    </message>
    <message>
        <location filename="../XTFLog_Checker_igcheck_dock_panel.py" line="63"/>
        <source>igCheck Error log</source>
        <translation>Journal des erreurs igCheck</translation>
    </message>
</context>
</TS>
